lib/node_modules/@stdlib/math/base/special/boxcoxinv/test/test.js

```diff
@@ -0,0 +1,147 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2018 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var randu = require( '@stdlib/random/base/randu' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var EPS = require( '@stdlib/constants/math/float64-eps' );
+var boxcoxinv = require( './../lib' );
+
+
+// FIXTURES //
+
+var mediumPositive = require( './fixtures/python/medium_positive.json' );
+var smallPositive = require( './fixtures/python/small_positive.json' );
+var tiny = require( './fixtures/python/tiny.json' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.equal( typeof boxcoxinv, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function returns `NaN` if provided `NaN`', function test( t ) {
+	var b;
+
+	b = boxcoxinv( NaN, 1.0);
+	t.equal( isnan( b ), true, 'returns NaN' );
+
+	b = boxcoxinv( 1.0, NaN );
+	t.equal( isnan( b ), true, 'returns NaN' );
+
+	b = boxcoxinv( NaN, NaN );
+	t.equal( isnan( b ), true, 'returns NaN' );
+
+	t.end();
+});
+
+tape( 'the function returns `NaN` if `x` is negative and `lambda` is one', function test( t ) {
```

Comment:
For the function to return `NaN`, does `lambda` have to be `1`? What if `lambda = 10`? Then, based on the implementation, I would assume that the function returns `NaN` for any combination of `y` and `lambda` which is less than negative one.

This test should be clarified.

Reply:
You are absolutely correct. I have, hopefully, addressed those concerns. Additionally, I added additionally testing for ranges of inputs. I recognize that I need to get better at testing so thank you for bringing that to my attention.